<ui version="4.0" >
 <class>picTypeDialog</class>
 <widget class="QDialog" name="picTypeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>pvQt - Picture Type and FOV</string>
  </property>
  <widget class="QWidget" name="layoutWidget" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>267</width>
     <height>204</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QLabel" name="label" >
      <property name="text" >
       <string>Source</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nameLabel" >
      <property name="frameShape" >
       <enum>QFrame::Box</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth" >
       <number>2</number>
      </property>
      <property name="text" >
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Picture type</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="typesBox" />
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" >
      <item>
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>Width, Height, pixels</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="widPixels" >
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth" >
         <number>2</number>
        </property>
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hgtPixels" >
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth" >
         <number>2</number>
        </property>
        <property name="text" >
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <item>
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>Max Field of View, degrees</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="fovBox" >
        <property name="accelerated" >
         <bool>true</bool>
        </property>
        <property name="minimum" >
         <double>5.000000000000000</double>
        </property>
        <property name="maximum" >
         <double>180.000000000000000</double>
        </property>
        <property name="value" >
         <double>180.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox" >
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons" >
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>picTypeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>picTypeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
